apcd-cms/src/apps/admin_extension/static/admin_extension/css/table.css

```diff
@@ -0,0 +1,43 @@
+.extension-table {
```

Comment:
Same thing here - can taccsite_custom table styles be used here instead of/in combo with a new stylesheet?